src/__tests__/components/fields/range-select/range-select.spec.tsx

```diff
@@ -0,0 +1,196 @@
+import { Button } from "@lifesg/react-design-system/button";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import { useState } from "react";
+import { FrontendEngine } from "../../../../components";
+import { IRangeSelectSchema } from "../../../../components/fields";
+import { IFrontendEngineData } from "../../../../components/frontend-engine";
+import {
+	ERROR_MESSAGE,
+	FRONTEND_ENGINE_ID,
+	TOverrideField,
+	TOverrideSchema,
+	getErrorMessage,
+	getField,
+	getSubmitButton,
+	getSubmitButtonProps,
+} from "../../../common";
+
+const SUBMIT_FN = jest.fn();
+const COMPONENT_ID = "field";
+const UI_TYPE = "range-select";
+
+const renderComponent = (overrideField?: TOverrideField<IRangeSelectSchema>, overrideSchema?: TOverrideSchema) => {
+	const json: IFrontendEngineData = {
+		id: FRONTEND_ENGINE_ID,
+		sections: {
+			section: {
+				uiType: "section",
+				children: {
+					[COMPONENT_ID]: {
+						label: "RangeSelect",
+						uiType: UI_TYPE,
+						options: {
+							from: [
+								{ label: "Apple", value: "Apple" },
+								{ label: "Berry", value: "Berry" },
+							],
+							to: [
+								{ label: "Cherry", value: "Cherry" },
+								{ label: "Date", value: "Date" },
```

Comment:
can make labels and values differ from each so it is easier to tell if the label or value is used
e.g. `{label: "Apple", value: "apple"}`